app/components/SessionNeedsReviewWarningComponent.rb

```diff
@@ -0,0 +1,30 @@
+class SessionNeedsReviewWarningComponent < ViewComponent::Base
```

Comment:
We prefix components with `App` to distinguish them from the standard `Govuk` components we use.